moto/resourcegroupstaggingapi/models.py

```diff
@@ -106,6 +108,20 @@ def lambda_backend(self):
         """
         return lambda_backends[self.account_id][self.region_name]
 
+    @property
+    def cloudformation_backend(self):
+        """
+        :rtype: moto.cloudformation.models.CloudFormationBackend
+        """
+        return cloudformation_backends[self.account_id][self.region_name]
```

Comment:
The CF-module imports all other modules, which means that the user needs to have all dependencies installed - which may not be the case

Can we try-catch the entire thing, just for CF? So that if anything fails with importing, we can still continue to collect tags for other services.
```
def get_cf_stack_tags():
    try:
        from moto.cloudformation import cloudformation_backends

        backend = cloudformation_backends[self.account_id][self.region_name]

        return all_tags
    except:
        return []
```

Reply:
updated, let me know if that's what you had in mind